pkg/config/app/kuma-dp/config.go

```diff
@@ -156,6 +156,8 @@ type DataplaneRuntime struct {
 	TokenPath string `yaml:"dataplaneTokenPath,omitempty" envconfig:"kuma_dataplane_runtime_token_path"`
 	// Token is dataplane token's value provided directly, will be stored to a temporary file before applying
 	Token string `yaml:"dataplaneToken,omitempty" envconfig:"kuma_dataplane_runtime_token"`
+	// DataplaneTemplate is a Dataplane resource that will be applied on Kuma CP
+	DataplaneTemplate string `yaml:"resource,omitempty" envconfig:"kuma_dataplane_runtime_resource"`
```

Comment:
`env` name should be in sync with YAML path

Reply:
It is the same now - "KUMA_" + "DATAPLANE_RUNTIME_" + "RESOURCE". Maybe the other resources in this struct have wrong names.